packages/ibm-products/src/components/ConditionBuilder/ConditionBuilderContent/ConditionBuilderContent.js

```diff
@@ -45,7 +44,7 @@ const ConditionBuilderContent = ({
   const onStartConditionBuilder = () => {
     //when add condition button is clicked.
     setIsConditionBuilderActive(false);
-    setRootState(emptyState); //here we can set an empty skeleton object for an empty condition builder,
+    setRootState(initialState ?? emptyState); //here we can set an empty skeleton object for an empty condition builder,
```

Comment:
We are giving an option to pass predefined state for a condition builder through the input prop(initialState). You can see the story 'With initial state' which is optional. Otherwise builder will start from the beginning (emptyState)

This is not connected with loading state.

Reply:
apologies i don't know why i confused the two in my head haha